src/internal/packager/images/push.go

```diff
@@ -46,7 +50,22 @@ func (i *ImgConfig) PushToZarfRegistry() error {
 	spinner := message.NewProgressSpinner("Storing images in the zarf registry")
 	defer spinner.Stop()
 
-	pushOptions := config.GetCraneAuthOption(i.RegInfo.PushUsername, i.RegInfo.PushPassword)
```

Comment:
Would it make sense to add the `i.Insecure` as a parameter to this function? I think there are other places that might benefit from having the roundTripper added to the options (such as `tools.go -> func zarfCraneCatalog`)

Reply:
`GetCraneAuthOption` should only provide auth. Insecure and round ripper can be different options as mentioned in [this](https://github.com/defenseunicorns/zarf/pull/1191#discussion_r1084374638) review by @Racer159

Reply:
I agree with @rishi-anand in keeping those separate for now.